pkgs/top-level/all-packages.nix

```diff
@@ -20749,6 +20749,17 @@ in
 
   syncthing-tray = callPackage ../applications/misc/syncthing-tray { };
 
+  syncthingtray-minimal = libsForQt5.callPackage ../applications/misc/syncthingtray { };
+  syncthingtray = libsForQt5.callPackage ../applications/misc/syncthingtray {
+    # Could be qt5.qtwebengine as well
+    webviewProvider = qt5.qtwebkit;
+    # Could be qt5.qtdeclarative as well
+    jsProvider = qt5.qtscript;
```

Comment:
Note that Qt Script support is only for people having fun experiencing strange JavaScript bugs :-)

Reply:
I've defaulted to `qtwebengine` and `qtdeclarative`.